plugin/errors/README.md

```diff
@@ -22,12 +22,12 @@ Extra knobs are available with an expanded syntax:
 
 ~~~
 errors {
-	consolidate DURATION REGEXP
+	consolidate DURATION REGEXP <LEVEL?>
```

Comment:
```suggestion
	consolidate DURATION REGEXP [LEVEL]...
```

Reply:
Also do we want multiple levels? Or do we assume the same error text will never be in multiple levels?

Reply:
I think it does not make sense to log the error message on multiple log levels. The error should be logged either as ERROR, WARNING, DEBUG, or INFO